FV3/atmos_cubed_sphere/model/fv_update_phys.F90

```diff
@@ -561,18 +563,27 @@ subroutine fv_update_phys ( dt, is, ie, js, je, isd, ied, jsd, jed, ng, nq,
         if (allocated(nudge_diag%nudge_u_dt)) nudge_diag%nudge_u_dt = u_dt(is:ie,js:je,:)
         if (allocated(nudge_diag%nudge_v_dt)) nudge_diag%nudge_v_dt = v_dt(is:ie,js:je,:)
 
+        if (.not. allocated(qv_dt_nudge)) allocate(qv_dt_nudge(is:ie,js:je,1:npz))
+        qv_dt_nudge = q(is:ie,js:je,:,sphum)
```

Comment:
Do we always want this new variable allocated and filled? or only when dycore_nudge is used? or does it not matter?

Reply:
Oh nevermind I see above it was suggested to make it standard behavior. I am fine with that!

Reply:
Yeah indeed I tried to be careful about this.  The way things are set up now, it actually is only allocated when nudging is active (this code is within a larger preexisting `if ( nudge ) then` block).  

As you noted, as per Oli's suggestion, we'll always make use of the specific humidity nudging tendency to address the land surface drift when nudging is active.  This makes it distinct from the other tendencies since we only track those if diagnostics are requested.

Reply:
Thanks for the comment though, because it made me realize there was an unneeded assignment statement leftover from the previous implementation of the specific humidity tendency calculation!  See aa5f2fe.